app/models/defence_request.rb

```diff
@@ -83,7 +83,7 @@ def own_solicitor?
   private
 
   def format_phone_number
-    self.phone_number = self.phone_number.gsub(/\D/, '')
+    self.phone_number = self.phone_number.gsub(/\D/, '') if self.phone_number
```

Comment:
Any reason we can't do this in a setter override and lose the callback?

```
def phone_number=(new_value)
  super(format_phone_number(new_value))
end

private

def format_phone_number(phone_number)
  phone_number.to_s.gsub(/\D/, '') if phone_number
end
```

Also are we sure that we want to enforce digits only on phone numbers?